Makefile

```diff
@@ -61,11 +61,11 @@ CXXFLAGS += -std=c++11 $(GRPC_GRPCPP_CFLAGS)
 ifeq ($(SYSTEM),Darwin)
 LDFLAGS += $(GRPC_GRPCPP_LDLAGS) \
            -lgrpc++_reflection -lgrpc_cronet \
-           -lprotobuf -lpthread -ldl -lcurl
```

Comment:
Is `lcurl` not needed?

Reply:
Well, I can definitely build and run the samples without cURL (on Ubuntu 16.04 and on diverse ARM-based boards). I don't know why it's there in the first place. I can't find any references to cURL API functions in the sources. cURL is no requirement for gRPC either. Putting `-lcurl` in the Makefile is misleading as it implies you need to have cURL available to use the SDK.

Reply:
If it builds without `-lcurl` then that's great.